### ISSUE TITLE
KeyError: "['close_26_ema'] not in index"

### ISSUE THREAD

**Issue by ltbd78** [jealous/stockstats#74]
Given any properly instantiated StockDataFrame, this snippet of code results in KeyError during first run, however works after second run (in Jupyter Notebook where variables are saved).
```
cols = ['open', 'close', 'high', 'low', 'volume', 'close_13_ema', 'close_26_ema', 'macd']
stock[cols]
```

When removing the ['close_26_ema'], there is no errors.
```
cols = ['open', 'close', 'high', 'low', 'volume', 'close_13_ema', 'macd']
stock[cols]
```

**Comment by jealous** [comment]
Hi,

`close_26_ema` is an intermediate result of `macd`.  After we calculated `macd`, the intermediate result is removed which gives you the key error.  To work around this issue, simply put `macd` ahead of `close_26_ema` in your colume list.

Thanks,
Cedric

**Comment by jealous** [comment]
Fixed in #105
